index.js

```diff
@@ -104,8 +104,6 @@ module.exports = function(S) {
 
       return functions.forEach(function(fun) {
         if( -1 !== fun.getRuntime().getName().indexOf('nodejs') ) {
-          // Override s-function.json defined environment!
-          Object.getPrototypeOf( fun.getRuntime() ).getEnvVars = ()=> BbPromise.resolve( {} );
```

Comment:
@tszajna0 I see what you mean but shouldn't the `serve` plugin work the same was as serverless core does?. In serverless core you can't override the variables defined in your s-function.json file. You need to properly configure your stage variables to make them work.